dashboard/controller/assignments/exportAssignment.js

```diff
@@ -46,6 +58,10 @@ module.exports = function exportAssignment(req, res) {
 		.query(q)
 		.end(function (error, response) {
 			if (response.statusCode == 200) {
+				// console.log(response.body.deliveries[0]);
+				// console.log(response.body.deliveries[0].content);
+				// console.log(response.body.deliveries[0].content[0]);
+				// console.log(response.body.deliveries[0].content[0].metadata);
```

Comment:
Please remove debug message.

Reply:
Oh!! yes , i forget to remove it. I will update the pr